haralyzer/assets.py

```diff
@@ -18,6 +18,14 @@
 
 DECIMAL_PRECISION = 0
 
+def get_entry_hostname(entry):
```

Comment:
Should we maybe move this to haralyzer.utils or something? Not a huge deal, just wondering.